test/type_test.js

```diff
@@ -83,4 +83,11 @@ describe('The plain.type function', function() {
     assertType(document.getElementsByTagName('script')[0],
         'object', 'other complex types');
   });
+
+  it('Identifies an arguments object', function() {
+    const argumentsTest = () => {
```

Comment:
Highly recommend changing this into an arrow function, for the same reason as above.